omnibus_overrides.rb

```diff
@@ -10,3 +10,6 @@
 override :rubygems, version: "2.6.8"
 # This SHA is the last commit before the 6.0 release
 override :'berkshelf-no-depselector', version: '6016ca10b2f46508b1b107264228668776f505d9'
+
+override :'postgresql-bin-versioned', version: "9.2.21" # needed for upgrading
```

Comment:
That's actually what was there before -- but they commenters convinced me that there's a value in having `postgres-bin-versioned` in omnibus-software, which can be used for any version we'd like to _upgrade from_. [However, that bit hasn't passed review yet either](https://github.com/chef/omnibus-software/pull/845)